dask/dataframe/io/csv.py

```diff
@@ -592,14 +592,18 @@ def read(
 read_fwf = make_reader(pd.read_fwf, "read_fwf", "fixed-width")
 
 
-def _write_csv(df, fil, **kwargs):
+def _write_csv(df, fil, result, **kwargs):
```

Comment:
This name is strange in the many csv files case.  I recommend `token` instead, which demotes the importance of this keyword for reviewers.